### PR TITLE
LEARNER-5070 Test - Android - Automate My Courses List screen/tab

### PR DESCRIPTION
**Pull request by nedo**
Followings features/stories are implemented,

* Verify that from Main Dashboard  tapping Courses tab will load My Courses contents(of specific logged in user) in its tab
* Verify that Courses tab/screen will show following header contents,,  
* Header Contents
** Profile icon
** "Courses" title  
** Account Icon 
* Courses Tab 
* Discovery Tab 
* Verify that My Courses(enrolled) List with followings in each course,
** Course image
** Course Name
** Course Start/End date
* "Looking for a new challenge?" label
* "Find a Course" button
* Add a global level function for Swipe through screen 
* Add a global level function to get child contents from list 
* Verify that tapping any course should load specific Course Dashboard screen 
* Verity that from Course Dashboard tapping back should load My Courses List screen 
* Verify that if no course is enrolled, there should be no courses list 
* Verify on tapping "Find a Course" button will load Discovery screen
* Verify that user should be able to scroll courses
* Implement Landscape support 
* Verify that user should be able to view all contents in Landscape mode 
* Verify that all functionality is working fine in Landscape mode

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review
cc @staubina

File: android/pages/android_my_courses_list.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            android_elements.my_courses_list_course_name
+        )
+
+        self.global_contents.get_elements_from_list(
+            self.driver,
+            android_elements.my_courses_list,
+            android_elements.my_courses_list_course_details
+        )
+
+        self.log.info('Total {} course rows are visible on screen. '.format(len(course_rows)))
```
Comment (raeeschachar) [inline android/pages/android_my_courses_list.py]: I am not able to understand why we need line 83 to 93
```self.global_contents.get_elements_from_list(
            self.driver,
            android_elements.my_courses_list,
            android_elements.my_courses_list_course_name
        )

        self.global_contents.get_elements_from_list(
            self.driver,
            android_elements.my_courses_list,
            android_elements.my_courses_list_course_details
        )
```
Can you please elaborate a bit?

File: android/tests/test_android_my_courses_list.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            set_capabilities.back()
+            assert android_main_dashboard_page.on_screen() == global_contents.MAIN_DASHBOARD_ACTIVITY_NAME
+            global_contents.swipe_screen(set_capabilities)
+
+        course_discovery_screen = android_my_courses_list_page.load_discovery_screen()
+        assert course_discovery_screen == global_contents.WEB_VIEW_FIND_COURSES_ACTIVITY_NAME
+        set_capabilities.back()
+        assert android_main_dashboard_page.on_screen() == global_contents.MAIN_DASHBOARD_ACTIVITY_NAME
+
+    def atest_landscape_smoke(self, set_capabilities, setup_logging):
```
[the remark this answers is not in the document]
Comment (nedo) [reply android/tests/test_android_my_courses_list.py]: done

File: android/pages/android_my_courses_list.py (hunk 1 above)
Comment (nedo) [reply android/pages/android_my_courses_list.py]: Basically each course has Name and Details(start/end date usually), first block of code is getting Course Name of all courses from list and second block is getting Course Details. I'll using that information further

**Comment by nedo** [comment]
@raeeschachar PR is ready for last pass

**Pull request merged**